src/components/TheFooter.vue

```diff
@@ -1,6 +1,10 @@
 <template>
   <footer class="c-footer">
-    <p class="c-footer__text">Thanks to all the Patrons!</p>
+    <p class="c-footer__text">
+      <BaseLink href="https://www.patreon.com/dylanbperry"
+        >Thanks to all the Patrons!</BaseLink
+      >
```

Comment:
I think this is slightly nicer formatting
```suggestion
      <BaseLink href="https://www.patreon.com/dylanbperry">
        Thanks to all the Patrons!
      </BaseLink>
```